test/extensions/voting-rep.js

```diff
@@ -213,7 +213,7 @@ contract("Voting Reputation", (accounts) => {
 
     const rootHash = await reputationTree.getRootHash();
     const repCycle = await getActiveRepCycle(colonyNetwork);
-    await forwardTime(MINING_CYCLE_DURATION, this);
+    await forwardTime(MINING_CYCLE_DURATION + SUBMITTER_ONLY_WINDOW + 1, this);
```

Comment:
Why is this needed? `SUBMITTER_WINDOW_ONLY` is only relevant during a dispute, which is not happening here.